include/picongpu/algorithms/FieldToParticleInterpolationNative.hpp

```diff
@@ -50,36 +50,41 @@ struct FieldToParticleInterpolationNative
     typedef typename pmacc::math::CT::make_Int<simDim,lowerMargin>::type LowerMargin;
     typedef typename pmacc::math::CT::make_Int<simDim,upperMargin>::type UpperMargin;
 
+    template<typename T_Type>
+    struct GetComponent {
```

Comment:
please check the code style `{` must be on a new line. Please check also the other new functors.